docs/quickstarts/insights-digital-roadmap/insights-digital-roadmap.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts 
+metadata:
+  name: insights-digital-roadmap
+  externalDocumentation: true
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+  displayName: Dynamically creating a digital roadmap to manage RHEL systems
+  icon: ~
+  description: Create tailored information by using RHEL Lightspeed Digital Roadmap to analyse your RHEL systems
```

Comment:
@elianerpereira for my own clarity - do we take a stance on using the British English (`analyse`) or American English (`analyze`) wording?